inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongTenantController.java

```diff
@@ -82,4 +83,13 @@ public Response<Boolean> update(@Validated(UpdateByIdValidation.class) @RequestB
         return Response.success(tenantService.update(request));
     }
 
+    @RequestMapping(value = "/tenant/delete/{name}", method = RequestMethod.GET)
+    @ApiOperation(value = "Delete inlong tenant by name")
+    @ApiImplicitParam(name = "name", dataTypeClass = String.class, required = true)
+    @RequiresRoles(logical = Logical.OR, value = {INLONG_ADMIN, INLONG_OPERATOR})
```

Comment:
INLONG_OPERATOR has no permission to do this.

Reply:
done